test/Microsoft.AspNetCore.Razor.Tools.Test/ServerCommandTest.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNetCore.Razor.Tools
 {
     public class ServerCommandTest
     {
-        [Fact]
+        [Fact(Skip = "https://github.com/aspnet/Razor/issues/2320")]
```

Comment:
This was fixed in dev which requires source changes. We know the cause of the flakiness - it's a test issue, so chosing to skip it.